kubetest2/internal/deployers/eksapi/infra.go

```diff
@@ -164,7 +183,10 @@ func (m *InfrastructureManager) deleteInfrastructureStack() error {
 			},
 			infraStackDeletionTimeout)
 	if err != nil {
-		return fmt.Errorf("failed to wait for infrastructure stack deletion: %w", err)
+		// don't fail the overall test, the janitor can clean this up
+		klog.Warningf("failed to wait for infrastructure stack deletion: %v", err)
+		m.metrics.Record(infraStackDeletionFailed, 1, nil)
+		return nil
```

Comment:
There any risk that this will cause things to look happy and continue our release without us releasing it's failed?

Reply:
Nope, this is just cleanup, it doesn't change anything about the test outcome. The janitor will take care of any leaks, and we don't want to fail the test because of a leaked ENI, for example.